diff --git a/.gitignore b/.gitignore
index 18555c4..b6dc4ab 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,4 +1,5 @@
 .env
+.env.prod
 .idea/
 node_modules/
 seeds.js
diff --git a/src/app.ts b/src/app.ts
index 7328c01..75cb4ff 100644
--- a/src/app.ts
+++ b/src/app.ts
@@ -169,8 +169,6 @@ app.use(
 						externalId: idToken.sub,
 						email: idToken.email,
 						registrationDate: Date.now(),
-						membership: 1209600000,
-						farmLimit: 1,
 						isProject: true,
 					});
 
@@ -183,8 +181,6 @@ app.use(
 				const newUser = await User.create({
 					externalId: idToken.sub,
 					registrationDate: Date.now(),
-					membership: 1209600000,
-					farmLimit: 1,
 					isProject: true,
 				});
 
diff --git a/src/migrations/remove-fields-2024-1.ts b/src/migrations/remove-fields-2024-1.ts
new file mode 100644
index 0000000..9eabd66
--- /dev/null
+++ b/src/migrations/remove-fields-2024-1.ts
@@ -0,0 +1,26 @@
+import User from "../models/user.js";
+
+import { connect } from "mongoose";
+
+try {
+	await connect(process.env.DATABASEURL as string); // CONNECTS TO MLAB MONGODB
+
+	const users = await User.find({});
+	for (const user of users) {
+		// user.farmLimit = undefined;
+		// user.haLimit = undefined;
+		// user.hash = undefined;
+		// user.username = undefined;
+		// user.salt = undefined;
+		// user.membership = undefined;
+		// user.resetPasswordExpires = undefined;
+		// user.resetPasswordToken = undefined;
+		// user.currentProject = undefined;
+
+		await user.save();
+	}
+} catch (error) {
+	console.error("Error fetching users:", error);
+}
+
+console.log("DONE");
diff --git a/src/models/user.ts b/src/models/user.ts
index 7407c99..bb496fd 100644
--- a/src/models/user.ts
+++ b/src/models/user.ts
@@ -6,18 +6,10 @@ import { IParcelSchema } from "./parcel.js";
 export interface IUserSchema {
 	externalId: string;
 	stripeCustomerId: string;
-	username: string;
-	password: string;
 	email: string;
-	resetPasswordToken: string;
-	resetPasswordExpires: Date;
 	parcels: [HydratedDocument<IParcelSchema>];
-	currentProject: HydratedDocument<IParcelSchema>;
 	countryCode: string;
 	registrationDate: number;
-	membership: number;
-	farmLimit: number;
-	haLimit: number;
 	isAdmin: boolean;
 	isNursery: boolean;
 	isManagement: boolean;
@@ -28,26 +20,16 @@ export interface IUserSchema {
 const UserSchema = new Schema({
 	externalId: { type: String, unique: true, require: false },
 	stripeCustomerId: { type: String, unique: true, require: false },
-	username: { type: String, unique: false, require: false },
-	password: String,
 	email: { type: String, unique: true, require: false },
-	resetPasswordToken: String,
-	resetPasswordExpires: Date,
 	parcels: [
 		{
 			type: Schema.Types.ObjectId,
 			ref: "Parcel",
 		},
 	],
-	currentProject: {
-		type: Schema.Types.ObjectId,
-		ref: "Parcel",
-	},
+	
 	countryCode: { type: String, default: "", require: false },
 	registrationDate: Number,
-	membership: Number,
-	farmLimit: Number,
-	haLimit: { type: Number, default: 5 },
 	isAdmin: { type: Boolean, default: false },
 	isNursery: { type: Boolean, default: false },
 	isManagement: { type: Boolean, default: false },
diff --git a/src/routes/activities.ts b/src/routes/activities.ts
index a5c98fe..0de18d2 100644
--- a/src/routes/activities.ts
+++ b/src/routes/activities.ts
@@ -198,7 +198,7 @@ router.delete(
 	) => {
 		// MAKE ACTIVITY OWNERSHIP MIDDLEWARE
 		try {
-			await Activity.findByIdAndRemove(req.params.id);
+			await Activity.findByIdAndDelete(req.params.id);
 			res.send("/activities");
 		} catch (err) {
 			console.log(err);
@@ -463,7 +463,7 @@ router.delete(
 					await updatedProject.save();
 					// DELETE ACTIVITY
 					try {
-						await Activity.findByIdAndRemove(req.params.pid);
+						await Activity.findByIdAndDelete(req.params.pid);
 						res.send(`/projects/${updatedProject._id}`);
 					} catch (err) {
 						console.log(err);
diff --git a/src/routes/areas.ts b/src/routes/areas.ts
index c4ac184..9a27a71 100644
--- a/src/routes/areas.ts
+++ b/src/routes/areas.ts
@@ -87,7 +87,7 @@ router.delete(
 				await updatedProject.save();
 				// DELETE ROW
 				try {
-					await Area.findByIdAndRemove(req.params.pid);
+					await Area.findByIdAndDelete(req.params.pid);
 					console.log(`Length after ${updatedProject.areas.length}`);
 					res.send(`/projects/${updatedProject._id}/layout`);
 				} catch (err) {
diff --git a/src/routes/assets.ts b/src/routes/assets.ts
index 8b3cbef..a87ff94 100644
--- a/src/routes/assets.ts
+++ b/src/routes/assets.ts
@@ -175,7 +175,7 @@ router.delete(
 
 							// DELETE ASSET
 							try {
-								await Asset.findByIdAndRemove(req.params.id);
+								await Asset.findByIdAndDelete(req.params.id);
 								res.send(`/projects/${foundProject._id}`);
 							} catch (err) {
 								console.log(err);
diff --git a/src/routes/index.ts b/src/routes/index.ts
index 3fb0433..91f158d 100644
--- a/src/routes/index.ts
+++ b/src/routes/index.ts
@@ -253,7 +253,7 @@ router.put(
 // router.delete("/users/:id", middleware.checkUserOwnership, async function(req: express.Request & { user?: UserDocument, idToken?: Auth0IDToken }, res: express.Response){
 //     try {
 
-//         let user = await User.findByIdAndRemove(req.params.id);
+//         let user = await User.findByIdAndDelete(req.params.id);
 
 //         // Flash message
 //         logger.info('User "' + user?.email + '" was deleted', {timestamp: Date.now()});
@@ -266,7 +266,7 @@ router.put(
 //     }
 // });
 
-// SET CURRENTPROJECT //
+
 
 router.put(
 	"/users/:id/countrycode",
@@ -293,35 +293,6 @@ router.put(
 	},
 );
 
-router.put(
-	"/users/:id/currentproject",
-	middleware.checkUserOwnership,
-	async (
-		req: express.Request & { user?: UserDocument; idToken?: Auth0IDToken },
-		res: express.Response,
-	) => {
-		console.log("im here 2");
-		try {
-			const foundUser = await User.findById(req.user?.id);
-			console.log("im here 3");
-			try {
-				const foundParcel = await Parcel.findById(req.body.parcelid);
-				console.log("im here 4", foundUser, foundParcel);
-				if (foundUser && foundParcel) {
-					console.log("im here 5");
-					foundUser.currentProject = foundParcel.id;
-					await foundUser.save();
-					console.log(`${foundParcel.name} has been set to active project`);
-					res.send(foundUser);
-				}
-			} catch (err) {
-				console.log(err);
-			}
-		} catch (err) {
-			console.log(err);
-		}
-	},
-);
 
 // PARCEL STATUS PAGE
 router.get(
diff --git a/src/routes/layers.ts b/src/routes/layers.ts
index 3d21821..3140f05 100644
--- a/src/routes/layers.ts
+++ b/src/routes/layers.ts
@@ -584,7 +584,7 @@ router.delete(
 					// REMOVE LAYER FROM PARCEL HERE WHEN IT IS FOUND?!
 					res.send();
 					// DELETE LAYER TEMP REMOVED
-					/* Layer.findByIdAndRemove(req.params.id, function(err){
+					/* Layer.findByIdAndDelete(req.params.id, function(err){
                        if(err){
                            console.log(err);
                            res.send("/parcels");
@@ -1317,7 +1317,7 @@ router.delete(
 				});
 				// DELETE ROW
 				try {
-					await Row.findByIdAndRemove(req.params.pid);
+					await Row.findByIdAndDelete(req.params.pid);
 
 					console.log(`Length after ${updatedLayer.rows.length}`);
 					res.send(`/layers/${updatedLayer._id}/layout`);
diff --git a/src/routes/parcels.ts b/src/routes/parcels.ts
index 24d62f2..a2eea46 100644
--- a/src/routes/parcels.ts
+++ b/src/routes/parcels.ts
@@ -237,7 +237,6 @@ router.post(
 					// Add the parcel to the users parcels for referencing
 					if (foundUser) {
 						foundUser.parcels.push(newlyCreated);
-						foundUser.currentProject = newlyCreated;
 						await foundUser.save();
 						// Save JSON file to geometry
 						// newlyCreated.geometry = req.body.geometry;
@@ -444,7 +443,7 @@ router.delete(
 		res: express.Response,
 	) => {
 		try {
-			await Parcel.findByIdAndRemove(req.params.id);
+			await Parcel.findByIdAndDelete(req.params.id);
 			res.send(`/users/${req.user?.id}`);
 		} catch (err) {
 			console.log(err);
diff --git a/src/routes/systems.ts b/src/routes/systems.ts
index 866b69a..84da4d6 100644
--- a/src/routes/systems.ts
+++ b/src/routes/systems.ts
@@ -977,7 +977,7 @@ router.delete(
 									}
 									// DELETE SYSTEM NOW
 									try {
-										await System.findByIdAndRemove(req.params.pid);
+										await System.findByIdAndDelete(req.params.pid);
 									} catch (err) {
 										console.log(err);
 									}
@@ -1041,7 +1041,7 @@ router.get(
 //           .exec();
 
 //         try {
-//           const foundParcel = await Parcel.findById(req.user?.currentProject);
+//           const foundParcel = await Parcel.findById(); // GET from URL
 
 //           if (foundParcel && foundSystem) {
 //             try {
@@ -1103,7 +1103,7 @@ router.get(
 //           .populate('animals')
 //           .exec();
 //         if (foundSystem) {
-//           const foundParcel = await Parcel.findById(req.user?.currentProject);
+//           const foundParcel = await Parcel.findById(); // GET from URL
 //           if (foundParcel) {
 //             // FIND SYSTEMS WITH SAME COMMODITY AS EXISTING SYSTEM (ONLY IF MONOCULTURE?) - COUNT OCCURRENCES?
 //             // let commodity = '';
